src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -83,18 +83,22 @@ const ProjectItemWrapper = styled.section`
 			padding: 1.5rem 1rem;
 			max-height: 125px;
 		}
-	}
+      @media screen and (max-width: 550px) {
+          img {
```

Comment:
Don't we wanr them to be specific to the `.proj_icon_2` class? Like the prevycode?

Reply:
It is inside the `.proj_icon_2` class

Reply:
Ohhh, I missed that curly brace 😂